packages/apps/plugins/plugin-client/src/ClientPlugin.tsx

```diff
@@ -77,7 +80,15 @@ export const ClientPlugin = ({
     initialize: async () => {
       let firstRun = false;
 
-      client = new Client({ config: new Config(await Envs(), Local(), Defaults()), ...options });
+      settings.prop(settings.values.$storageDriver!, 'storage-driver');
+      if (!settings.values.storageDriver && (await defaultStorageIsEmpty())) {
+        settings.values.storageDriver = defs.Runtime.Client.Storage.StorageDriver.IDB;
+      }
+
+      client = new Client({
+        config: new Config(await Envs(), Local(), Defaults(), configFromSettings(settings.values)),
```

Comment:
note that if you want this to override other settings it should go at the beginning

Reply:
also, we can actually remove wait from Envs, its not async